blazor/datagrid/column-resizing.md

```diff
@@ -7,143 +7,366 @@ control: DataGrid
-        public double? Freight { get; set; }
+        public string ShipCity { get; set; }
+        public string ShipName { get; set; }
+        public string ShipCountry { get; set; }
+        public string ShipAddress { get; set; }
+        public double Freight { get; set; }
     }
-}
-```
+{% endhighlight %}
+{% endtabs %}
 
-The following GIF represents column resizing, 
+{% previewsample "https://blazorplayground.syncfusion.com/embed/VZhKsCXsTYapJQWa?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
 
-![Resizing Column in Blazor DataGrid](images/blazor-datagrid-column-resizing.gif)
-
-N> You can disable resizing for a particular column by setting the [AllowResizing](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_AllowResizing) property of **GridColumn** component to false.
+> You can disable resizing for a particular column by setting the [AllowResizing](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_AllowResizing) property of **GridColumn** component to false.
 <br/> In RTL mode, you can click and drag the left edge of the header cell to resize the column.
+<br/>The width property of the column can be set initially to define the default width of the column. However, when column resizing is enabled, you can override the default width by manually resizing the columns.
 
-## Min and max width
+## Restrict the resizing based on minimum and maximum width
 
-Column resize can be restricted between minimum and maximum width by defining the [MinWidth](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_MinWidth) and [MaxWidth](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_MaxWidth) properties in  **GridColumn** component.
+The Grid component allows you to restrict the column width resizing between a minimum and maximum width. This can be useful when you want to ensure that your grid’s columns stay within a certain range of sizes.
 
-In the following sample, minimum and maximum width are defined for **CustomerID** and **Freight** columns.
+To enable this feature, you can define the [MinWidth](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_MinWidth) and [MaxWidth](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_MaxWidth) properties of the columns directive for the respective column.
 
-```cshtml
+In the below code, **CustomerID** and **Freight** columns are defined with minimum and maximum width. The CustomerID column is set to have a minimum width of 10 pixels and a maximum width of 200 pixels. Similarly, the Freight column is set to have a minimum width of 10 pixels and a maximum width of 200 pixels. 
```

Comment:
in code example one column is used. but here different column names were specified. check in sample too

Reply:
modified